Kickstarter-iOS/Views/Cells/PledgeDescriptionCell.swift

```diff
@@ -0,0 +1,155 @@
+
+    _ = self.rootStackView
+      |> rootStackViewStyle
+
+    _ = self.containerImageView
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+      |> \.backgroundColor .~ UIColor.blue
+
+    _ = self.spacerView
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+
+    _ = self.pledgeImage
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+      |> \.backgroundColor .~ UIColor.orange
+
+    _ = self.descriptionStackView
+      |> descriptionStackViewStyle
+
+    _ = self.estimatedDeliveryLabel
+      |> estimatedDeliveryLabelStyle
+
+    _ = self.dateLabel
+      |> dateLabelStyle
+
+    _ = self.descriptionLabel
+      |> descriptionLabelStyle
+
+    _ = self.learnMoreLabel
+      |> learnMoreLabelStyle
+  }
+
+  private func arrangeStackView() {
+    self.spacerView.heightAnchor.constraint(equalToConstant: 10.0).isActive = true
+    self.descriptionStackView.addArrangedSubview(self.spacerView)
+    self.descriptionStackView.addArrangedSubview(self.estimatedDeliveryLabel)
+    self.descriptionStackView.addArrangedSubview(self.dateLabel)
+    self.descriptionStackView.addArrangedSubview(self.descriptionLabel)
+    self.descriptionStackView.addArrangedSubview(self.learnMoreLabel)
+
+    if #available(iOS 11.0, *) {
+      self.descriptionStackView.setCustomSpacing(10.0, after: self.dateLabel)
+    } else {
+      let view = UIView(frame: .zero)
+      view.heightAnchor.constraint(equalToConstant: 10.0).isActive = true
+      self.descriptionStackView.insertArrangedSubview(view, at: 3)
+    }
+
+    self.rootStackView.addArrangedSubview(self.descriptionStackView)
+  }
+}
+
+private let rootStackViewStyle: StackViewStyle = { (stackView: UIStackView) in
+  stackView
+    |> \.alignment .~ .top
+    |> \.axis .~ .horizontal
+    |> \.translatesAutoresizingMaskIntoConstraints .~ false
+    |> \.isLayoutMarginsRelativeArrangement .~ true
+    |> \.layoutMargins .~ UIEdgeInsets.init(topBottom: Styles.grid(5), leftRight: Styles.grid(2))
+    |> \.spacing .~ Styles.grid(2)
+}
+
+private let descriptionStackViewStyle: StackViewStyle = { (stackView: UIStackView) in
+  stackView
+    |> \.axis .~ .vertical
+    |> \.distribution .~ .fill
+}
+
+private let estimatedDeliveryLabelStyle: LabelStyle = { (label: UILabel) in
+  label
+    |> \.text %~ { _ in Strings.Estimated_delivery_of() }
+    |> \.textColor .~ UIColor.ksr_text_dark_grey_500
+    |> \.font .~ UIFont.ksr_headline()
+    |> \.adjustsFontForContentSizeCategory .~ true
+    |> \.numberOfLines .~ 0
+}
+
+private let dateLabelStyle: LabelStyle = { (label: UILabel) in
+  label
+    |> \.textColor .~ UIColor.ksr_soft_black
+    |> \.font .~ UIFont.ksr_headline()
+    |> \.adjustsFontForContentSizeCategory .~ true
+    |> \.numberOfLines .~ 0
+}
+
+private let descriptionLabelStyle: LabelStyle = { (label: UILabel) in
+  label
+    |> \.text  %~ { _ in "Kickstarter is not a store. It's a way to bring creative projects to life." }
```

Comment:
If we don't have a localized string yet then it's a good idea to use `localizedString` directly:

```swift
localizedString(
  key: "Kickstarter_is_not_a_store_Its_a_way_to_bring_creative_projects_to_life", 
  defaultValue: "Kickstarter is not a store. It's a way to bring creative projects to life."
)
```

That way we can just search for `localizedString` to know which ones we still need to replace and it also already invokes the localization.

Reply:
There's a PR for this string opened now, I will update this as soon as it's available